docker-multinode/common.sh

```diff
@@ -123,40 +125,12 @@ kube::multinode::detect_lsb() {
   kube::log::status "Detected OS: ${lsb_dist}"
 }
 
-# Start a docker bootstrap for running etcd and flannel
-kube::multinode::bootstrap_daemon() {
```

Comment:
I assume the CNI solution will be fully available from `v1.4-` This PR makes sense when we remove `docker bootstrap` Do you want keep booth solutions ?